tests/Feature/app/Http/Controllers/HealthControllerTest.php

```
@@ -33,22 +35,20 @@ public function it_checks_if_up_endpoint_returns_operational_status(): void
+                'laravel_version',
             ])
             ->assertJson([
                 'status' => 'up',
-                'message' => 'API is operational'
+                'message' => 'API is operational',
             ]);
 
         // Verify timestamp is valid ISO 8601
         $data = $response->json();
-        $this->assertNotNull(\DateTime::createFromFormat(\DateTime::ISO8601, $data['timestamp']));
+        $this->assertNotNull(DateTimeImmutable::createFromFormat(DateTime::ISO8601, $data['timestamp']));
```

PHPMD (2.15.0)

Avoid using static access to class '\DateTimeImmutable' in method 'it_checks_if_up_endpoint_returns_operational_status'. (undefined)

(StaticAccess)